ds3-sdk/src/test/java/com/spectralogic/ds3client/metadata/MACMetadataRestore_Test.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * ****************************************************************************
+ *    Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *    Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *    this file except in compliance with the License. A copy of the License is located at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    or in the "license" file accompanying this file.
+ *    This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *    CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *    specific language governing permissions and limitations under the License.
+ *  ****************************************************************************
+ */
+package com.spectralogic.ds3client.metadata;
+
+import com.google.common.collect.ImmutableMultimap;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import com.spectralogic.ds3client.commands.interfaces.MetadataImpl;
+import com.spectralogic.ds3client.metadata.interfaces.MetadataRestoreListener;
+import com.spectralogic.ds3client.networking.Headers;
+import com.spectralogic.ds3client.networking.Metadata;
+import com.spectralogic.ds3client.utils.MetaDataUtil;
+import com.spectralogic.ds3client.utils.MetadataKeyConstants;
+import org.apache.http.Header;
+import org.apache.http.message.BasicHeader;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.List;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public class MACMetadataRestore_Test {
+
+    private final File file = new File(getClass().getClassLoader().getResource("LoremIpsum.txt").getFile());
```

Comment:
This test failed on my mac

org.junit.ComparisonFailure: 
Expected :1484248020000
Actual   :1484248028000